packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -30,6 +30,59 @@ import 'tooltip_theme.dart';
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=EeEfD5fI-5Q}
 ///
+/// ## Example
+///
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// This example show a basic [Tooltip] which has a [Text] as child.
+/// [message] contains your label to be shown on long pressed.
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   return Center(
+///     child: Tooltip(
+///       message: "Welcome Greetings",
+///       child: Text("Hello Flutter"),
```

Comment:
Might be helpful to have the text itself better describe what the user should expect from the code sample. Otherwise, it could be confusing to just see a Text widget without the tooltip.
```suggestion
///       child: Text("Tap this text and hold down to show a tooltip."),
```